app/forms/idv/send_phone_confirmation_otp_form.rb

```diff
@@ -0,0 +1,106 @@
+module Idv
+  # :reek:InstanceVariableAssumption
+  class SendPhoneConfirmationOtpForm
+    include ActiveModel::Model
+
+    attr_accessor :user, :idv_session, :locale
+
+    validates :otp_delivery_preference, inclusion: { in: %i[sms voice] }
```

Comment:
I think we can remove this since the preference was already validated in the previous step.

Reply:
Yeah, there's some slippery spots in the code that result from trying to validate this. I'll pull them out. I'll probs raise for an invalid delivery preference instead.